drivers/regulator/Kconfig.nxp_vref

```diff
@@ -12,3 +12,10 @@ config REGULATOR_NXP_VREF_INIT_PRIORITY
 	default 45
 	help
 	  Init priority for the NXP VREF peripheral.
+
+config REGULATOR_NXP_VREF_HAS_REFL_GRD_SEL
+	bool "NXP VREF peripheral has ground select"
+	default n if SOC_MCXN947
+	default y if SOC_LPC55S36
+	help
+	  Decides whether NXP VREF peripheral has ground select.
```

Comment:
No need for this as the suggestion from the VREF designer is to delete the use of `CSR_REFL_GRL_SEL` bit